automation_tools/__init__.py

```diff
@@ -2127,6 +2130,151 @@ def get_hostname_from_ip(ip, timeout=3):
     return output[ip]
 
 
+def get_rhevm_client():
+    """Creates client object instance for rhevm.
+
+    Returns the client object for further use.
```

Comment:
Remove this information and state that the caller of this function should take care of closing the client.